manifests/params.pp

```diff
@@ -62,7 +62,10 @@
     $server_root          = '/etc/httpd'
     $conf_dir             = "${httpd_dir}/conf"
     $confd_dir            = "${httpd_dir}/conf.d"
-    $mod_dir              = "${httpd_dir}/conf.d"
+    $mod_dir              = $::apache::version::distrelease ? {
+      '7'     => "${httpd_dir}/conf.modules.d",
+      default => "${httpd_dir}/conf.d",
```

Comment:
If you want to talk consistency, then look further down in the file (line 101) where it's using the same syntax I'm using.